### PR TITLE
API Updates for beta branch

### PR DESCRIPTION
**Pull request by dcr-stripe**
Codegen for openapi v170.
r? @kamil-stripe
cc @stripe/api-libraries

## Changelog
- Updated stable APIs to the latest version
- Add `QuotePhase` resource

### CODE REVIEW

**Review by kamil-stripe — APPROVED** [review]

File: types/2020-08-27/QuotePhases.d.ts [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+      /**
+       * Total after discounts and taxes are applied.
+       */
+      amount_total: number;
+
+      /**
+       * If set to `new`, the billing_cycle_anchor of the subscription is set to the start of the phase when entering the phase. If unset, then the billing cycle anchor is automatically modified as needed when entering the phase. For more information, see the billing cycle [documentation](https://stripe.com/docs/billing/subscriptions/billing-cycle).
+       */
+      billing_cycle_anchor: 'reset' | null;
```
Comment (kamil-stripe) [inline types/2020-08-27/QuotePhases.d.ts:32]: How can this be `new`?

File: types/2020-08-27/SubscriptionItems.d.ts [hunk 2]
```diff
@@ -261,6 +300,33 @@ declare module 'stripe' {
...
+        /**
+         * Details of a different price, quantity, or both, to bill your customer for during a paid trial.
+         */
+        paid?: Trial.Paid;
+
+        type: Trial.Type;
+      }
+
+      namespace Trial {
+        interface Free {}
```
Comment (kamil-stripe) [inline types/2020-08-27/SubscriptionItems.d.ts:318]: This is weird too

Comment (dcr-stripe) [reply types/2020-08-27/SubscriptionItems.d.ts:318]: This is expected - it's a polymorphic value where based on the type, you'll have some extra type-specific properties (eg. see the `Paid` interface below). Basically this is a "one of".

File: types/2020-08-27/QuotePhases.d.ts (hunk 1 above)
Comment (dcr-stripe) [reply types/2020-08-27/QuotePhases.d.ts:32]: Going to follow up in a second PR. Great catch!

**Review by kamil-stripe — APPROVED** [review]

**Pull request merged**